hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -45,7 +45,8 @@ staking.maxStakeRewarded=5000000000000000000
 staking.periodMins=1
 staking.rewardBalanceThreshold=0
 staking.rewardHistory.numStoredPeriods=365
-staking.startupHelper.recompute=NODE_STAKES,PENDING_REWARDS
+# Comma-separated list of { NODE_STAKES, PENDING_REWARDS }
+staking.startupHelper.recompute=
```

Comment:
Is there a config record default value that also needs to be updated for the modular code?